_content/development.md

```diff
@@ -0,0 +1,48 @@
+
+Project teams usually include: a *scrum master*, who ensures that nothing is impeding the work of the development team; a *client liaison*, who ensures that the client is kept fully informed of progress on the project; a *lead developer*, who ensures code quality; and a *product owner*, who takes responsibility for approving user stories as they are delivered. The product owner is usually supplied by the client (for whom we can supply training, if required).
+
+Change requests are added to the *project backlog* by the product owner and incorporated as a matter of course through the use of a repository of new user stories (the *“icebox”*) and a process of re-prioritisation of user stories at the beginning of every sprint. Minor changes are expected and can usually be accommodated within previously-agreed time and budget.
+
+## Test-driven development
+
+We practice *test-driven development* (TDD) and *behaviour-driven development* (BDD), where product features, expressed as user stories, are turned into *acceptance tests*, i.e. automated software tests that demonstrate whether or not the application software actually behaves as expected, and these are written before any code is actually created. We use automated tools to maintain measures of code coverage (i.e. what proportion of the code is covered by an associated test) and to identify failing tests.  
+
+We practice *paired programming* to ensure that all developers involved in a project share a common approach to the codebase and create readable code.  We undertake *code reviews* where developers not involved in a project review the quality of the code being created by the development team.
+
+We deliver well-documented code, based around concise README documents, so that in years to come our code will be maintainable, either by us or by others, and that anyone returning to the codebase or looking at it with fresh eyes will be able to understand our original intent. 
+
+## Training
+
+We offer training and support in *product management* to our clients. We particularly recommend our low-cost prototyping service, which provides the opportunity to combine valuable hands-on training with real-world application specification and product development.
+
+## Our technology stack
+  
++ Languages: JavaScript 
++ Backend: Node.js, hapi, Express, Sails.js
++ Frontend: React, AngularJS, Ampersand.js
++ Mobile: React Native, PhoneGap, Ionic
++ Data visualisation and mapping: D3, Leaflet 
++ Deployment: GitHub, Docker, Vagrant, Heroku, Amazon S3, Microsoft Azure 
++ Databases: MongoDB, PostgresSQL, Redis, ElasticSearch
++ Testing: QUnit, Mocha, Lab, Tape, Jasmine, Protractor
```

Comment:
I would move testing higher up as it's quite important to our philosophy (I feel!)